diamond_agent/tests/resources/blueprint/multi_node.yaml

```diff
@@ -25,7 +25,6 @@ node_templates:
                         diamond_config: { get_input: diamond_config }
                 start: diamond.diamond_agent.tasks.start
                 stop: diamond.diamond_agent.tasks.stop
-                uninstall: diamond.diamond_agent.tasks.uninstall
```

Comment:
I now see it is indeed problematic.
Lets remove the uninstall as you previously did but add a comment to the section in the test specifying the it specifically tests the `stop` operation logic